salt/utils/dictupdate.py

```diff
@@ -50,7 +54,7 @@ def update(dest, upd, recursive_update=True, merge_lists=False):
             elif isinstance(dest_subkey, list) \
                      and isinstance(val, list):
                 if merge_lists:
-                    dest[key] = dest.get(key, []) + val
+                    dest[key] = sorted((set(dest_subkey + val)))
```

Comment:
Yeah, that was my backup plan. Personally, it feels like we are modifying the order anyway if we remove duplicates, and I think we should be _explicit_ when merging lists about the resulting order rather than relying on an implementation detail (list comprehension vs set). Would a better approach be to gate this behind a new param for `dictupdate.update`?